### DISCUSSION TITLE
After completion of course even when user not meeting the certificate criteria still the criteria met status is showing yes to content creator

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#638] [General Discussion]
We are using the 5.0 version and testing it.
We have created one course with an Assessment for merit certificates.
The criteria for the Assessment or Merit certificate have been set to 90%.
Even after the user scores below 90% in the assessment which means the user does not meet the set criteria for assessment or merit certificate 
still the system is showing "**YES**" on the dashboard as the criteria have been met.
**Please help us understand this scenario and help us to fix this issue as soon as possible as we are stuck here and could not found any solution**
Check the screenshot for reference
![course_criteria](https://github.com/Sunbird-Ed/Community/assets/109504832/8076b777-e651-4802-b216-d759facebec7)
![course_result_for_criteria](https://github.com/Sunbird-Ed/Community/assets/109504832/f670fcc7-1b4b-4263-a26d-641e6d99ef63)
![image](https://github.com/Sunbird-Ed/Community/assets/109504832/3f95bf4f-a5b0-4901-95a1-4aee23fcc8c1)

@vinukumar-vs @rajeevsathish @rajeshkumaravel @Krishnaj20

**Comment by vinukumar-vs** [comment]
@sachinb80 Just by attaching the snapshots of UI is not good enough to share the suggestions. Because complete certificate generation logic is depends on backend/Flink process. 

I would suggest you to share your analysis or details of this specific user against the batch.
Please refer the below microsite for more details of cert generation
https://lern.sunbird.org/use/developer-guide/lms-service/certificate-flow

[@AmiableAnil](https://github.com/AmiableAnil), @gandham-santhosh

**Reply by rajeevsathish** [reply]
@sachinb80 please tell us what is the criteria that has been added to the batch please specify this and then see why the backend is saying the criteria is satisfied.

cc: @vinukumar-vs @Krishnaj20 @ashokreddy1208

**Reply by sachinb80** [reply]
Sure

**Comment by ashokreddy1208** [comment]
@sachinb80 Can you share required details to look into this issue as we are unable to reproduce this issue from our end.

**Reply by sachinb80** [reply]
Ok i will share the steps how to reproduce the issue @ashokreddy1208

**Comment by ashokreddy1208** [comment]
@sachinb80  Can you please share your analysis for this specific user against the batch.

**Reply by sachinb80** [reply]
sure

**Reply by sachinb80** [reply]
@ashokreddy1208 @rajeevsathish here is the video in which i have created the batch and set the criteria and consumed the course with user and showed the before and after status of certificate criteria. Please check and me know if you guys need any more details. Please checkout the link below for the video. **Sorry for the 8 min long video.**
https://app.flonnect.com/view/video/sachin_b/Flonnect_2023-10-17_0b35667d-5595-4535-ba9b-9155455f14c4

**Reply by Krishnaj20** [reply]
@sachinb80 @coolbung - It will be difficult for community to provide advise basis the video. We need specific debug analysis, and where you are stuck in your analysis. The microsite link shared by @vinukumar-vs has details of lern microservices which enable credentials. Request you to go thru it and carry out the debugging. 
@vinod303 - can you help with the debugging for @sachinb80 please, and help him get the necessary logs and information which can help community advise on how to resolve this issue?

@vinukumar-vs @amiableanil

**Reply by sachinb80** [reply]
Sure @Krishnaj20
